HTML/mockup-corporate/index.html

```diff
@@ -0,0 +1,329 @@
+                <source media="(min-width: 1500px)" srcset="img/partner-logo-7@3x.png">
+                <source media="(min-width: 500px)" srcset="img/partner-logo-7@2x.png">
+                <img src="img/partner-logo-7.png" alt="partner logo" class="section-people__logo">
+            </picture>
+            <picture>
+                <source media="(min-width: 1500px)" srcset="img/partner-logo-8@3x.png">
+                <source media="(min-width: 500px)" srcset="img/partner-logo-8@2x.png">
+                <img src="img/partner-logo-8.png" alt="partner logo" class="section-people__logo">
+            </picture>
+        </div>
+    </section>
+
+    <section class="container section-awesome">
+        <div class="section-awesome__content">
+            <p class="section-heading section-heading--label">NEW FEATURES</p>
+            <h2 class="section-heading">
+                Some awesome features
+            </h2>
+            <p class="section-awesome__paragraph paragraph">
+                Aenean eu leo quam. Pellentesque ornare sem lacinia quam venenatis vestibulum. Donec id elit non mi
+                porta gravida at eget metus. Curabitur blandit tempus porttitor. Etiasem malesuada magn. Vestibulum
+                felis euismod semper.
+            </p>
+        </div>
+        <div class="section-awesome__picture">
+            <picture>
+                <source media="(min-width: 1500px)" srcset="img/rectangle-24@3x-min.png">
+                <source media="(min-width: 500px)" srcset="img/rectangle-24@2x-min.png">
+                <img src="img/rectangle-24-min.png" alt="browser picture">
+            </picture>
+        </div>
+    </section>
+
+    <section class="container section-form">
+        <div class="section-form__content">
+            <p class="section-heading section-heading--label">NEW FEATURES</p>
+            <h2 class="section-heading u-color-white">
+                Over 1000 designers are using ...
+            </h2>
+        </div>
+
+        <form class="section-form__form form">
+            <div class="form__info-trial">
+                <picture>
+                    <source media="(min-width: 1500px)" srcset="img/arrow@3x.png">
+                    <source media="(min-width: 500px)" srcset="img/arrow@2x.png">
+                    <img src="img/arrow.png" alt="arrow icon">
+                </picture>
+                <p class="paragraph paragraph--grey">30 days free trial for all.</p>
+            </div>
+            <input class="form__input input" type="text" placeholder="FULL NAME">
+            <input class="form__input input" type="email" placeholder="YOUR EMAIL">
+            <input class="form__input input" type="password" placeholder="PASSWORD">
+            <button class="form__btn btn">TRY NOW</button>
+        </form>
+        <p class="section-form__terms paragraph paragraph--grey">By Signing up you agree to our <a href="#">terms &
+                Services.</a>
+        </p>
```

Comment:
it will be more readable
```html
<p class="section-form__terms paragraph paragraph--grey">
    By Signing up you agree to our <a href="#">terms &Services.</a>
</p>
```
or
```html
<p class="section-form__terms paragraph paragraph--grey">
    By Signing up you agree to our
    <a href="#">terms &Services.</a>
</p>
```